spec/models/user_spec.rb

```diff
@@ -4,6 +4,15 @@
   describe "validations" do
     it { should validate_presence_of(:name) }
     it { should validate_presence_of(:email) }
+
+    it "should not allow an email to be changed" do
+      user = create(:administrator)
+
+      user.email = "new@example.com"
+
+      expect(user.valid?).to eq(false)
```

Comment:
You could also write: `expect(user).to_not be_valid` or even `expect(user).to be_invalid`